### ISSUE TITLE
Mermaid mindmap

### ISSUE THREAD

**Issue by definetly-not-carl** [marktext/marktext#3532]
Hi there, i would like to use the mindmap feature for mermaid, but it doesn't render and i don't know how to import it in MarkText.
Does anybody wether it's possible to insert it or not?

**Comment by Huihh** [comment]
minmap and gitgraph doesn't seem to be supported at marktext v0.17.1

**Comment by wang1212** [comment]
You just need to place the cursor on a blank line, and then select the button on the left to format, as shown in the figure below:

![image](https://user-images.githubusercontent.com/18679119/211727498-1d8516c5-5eea-462c-8326-46e43043f92e.png)

**Comment by definetly-not-carl** [comment]
> You just need to place the cursor on a blank line, and then select the button on the left to format, as shown in the figure below:
> 
> ![image](https://user-images.githubusercontent.com/18679119/211727498-1d8516c5-5eea-462c-8326-46e43043f92e.png)
> 

Yes, I know how to create a basic mermaid graph, I meant a mindmap like this https://mermaid.js.org/syntax/mindmap.html

**Comment by Yuang114514** [comment]
so is there any plan for supporting this???

**Comment by Jocs** [comment]
Thanks for the request, and sorry for the long silence! When this was filed, MarkText (v0.17.x) bundled a much older Mermaid that predated mindmap support. The current engine now ships Mermaid 11.x and renders Mermaid code generically — no per-diagram allowlist — so `mindmap` (and `gitGraph`) diagrams render out of the box. Just put your cursor on a blank line, insert a Mermaid diagram block, and paste your mindmap syntax from https://mermaid.js.org/syntax/mindmap.html.

Closing as resolved by the engine/dependency upgrade. If a specific mindmap still fails to render on a current build, please reopen with the exact diagram code and we'll take another look!
